etc/wireshark.profile

```diff
@@ -12,7 +12,7 @@ include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
-# caps.drop all
+caps.keep net_admin,net_raw
```

Comment:
On my system it needs `dac_override` as well.

Reply:
Thank you! I will add it.